src/main/java/kr/codesqaud/cafe/service/ArticleService.java

```diff
@@ -1,32 +1,40 @@
 package kr.codesqaud.cafe.service;
 
-import kr.codesqaud.cafe.Repository.ArticleRepository;
+import kr.codesqaud.cafe.Repository.article.ArticleRepository;
 import kr.codesqaud.cafe.domain.Article;
+import kr.codesqaud.cafe.dto.article.ArticleRequestDto;
+import kr.codesqaud.cafe.dto.article.ArticleResponseDto;
 import org.springframework.stereotype.Service;
 
+import java.util.ArrayList;
 import java.util.List;
 
 @Service
 public class ArticleService {
     private final ArticleRepository articleRepository;
 
-    ArticleService(ArticleRepository articleRepository) {
-        this.articleRepository = new ArticleRepository();
+    ArticleService(final ArticleRepository articleRepository) {
+        this.articleRepository = articleRepository;
     }
 
-    public Article saveArticle(Article article) {
-        articleRepository.saveArticle(article);
-        return article;
+    public void saveArticle(final ArticleRequestDto articleRequestDto) {
+        articleRepository.saveArticle(articleRequestDto.toEntity());
     }
 
-    public List<Article> findArticles() {
-        return articleRepository.findAllArticle();
+    public List<ArticleResponseDto> findArticles() {
+        List<ArticleResponseDto> articleResponseDtos = new ArrayList<>();
+        List<Article> articles = articleRepository.findAllArticle();
+        for(Article article: articles){
```

Comment:
```suggestion
        for (Article article: articles){
```

이런 포매팅은 자동화 도구를 이용하면 좋을 것 같아요~ 한 번 알아보시길 바랍니다.

Reply:
습관화해서 사용하지 않다보니 지저분한 코드가 남발된 것 같습니다. 
학습 후에 습관적으로 적용해보도록 하겠습니다!